confdgnmi/tests/client_server_test_base.py

```diff
@@ -63,7 +63,12 @@ def capability_supported(cap):
 
         # check if selected capabilities are supported
         for cap in GnmiDemoServerAdapter.capability_list:
-            assert (capability_supported(cap))
+            assert capability_supported(cap)
+
+        # JSON - explicit value from gnmi.proto - mandatory according to standard
+        assert 0 in capabilities.supported_encodings
```

Comment:
use enum values, not numbers gnmi_pb2.Encoding.JSON

Reply:
replaced with enum values - wanted to reduce dependency on low level lib usage by using the proto/standard values